.vsts-ci.yml

```diff
@@ -53,11 +53,11 @@ stages:
       agentOs: Windows_NT
       pool:
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          name: NetCorePublic-Pool
-          queue: buildpool.windows.10.amd64.vs2019.open
+          name: NetCore1ESPool-Svc-Public
```

Comment:
This is a R&D branch and you're trying to use servicing pools

Reply:
embarrassing